usi/src/main/java/switchtest/USIImpl.java

```diff
@@ -0,0 +1,92 @@
+package switchtest;
+
+import grpc.*;
```

Comment:
I don't think we ever converged on the "why grpc" question in the design proposal -- it seems to have been deleted without resolution. Also the design proposal was never finalized (still indicated as draft). I want to understand "why grpc" here before proceeding... it's a significant bit of complexity for something that could really be simple. (If we already had a conversation about it please remind me.)  I'm not saying it's *wrong*, I just want to make sure we're properly reviewing/discussing things before executing on them (it's a procedural thing).

Reply:
I remember seeing a comment related to grpc in my email but I couldn't find the comment when I went to the design doc. I don't know any good alternatives for this purpose?